buildSrc/src/main/groovy/io/vantiq/extsrc/assy/tasks/AssemblyResourceGeneration.groovy

```diff
@@ -251,6 +253,13 @@ class AssemblyResourceGeneration extends DefaultTask {
         Map<String, Object> yamlMap = getYamlEnt(yamlLoader, classLoader, kameletDefinition)
         log.debug('    \n<<<Result: parsed.size(): {}, parsed: {}>>>\n', yamlMap.size(),
             yamlMap.get('spec'))
```

Comment:
check labels as defined to make sure they match the naming conventions we're using.